code/game/objects/structures/window.dm

```diff
@@ -265,10 +265,9 @@
 		else
 			icon_state = "[basestate][junction]"
 
-/obj/structure/window/fire_act(exposed_temperature, exposed_volume)
-	if(exposed_temperature > T0C + 800)
-		take_damage(round(exposed_volume / 100), BURN, FIRE)
-	return ..()
+/obj/structure/window/fire_act(burn_level)
+	if(burn_level > 25)
+		take_damage(burn_level, BURN, FIRE)
```

Comment:
Do you wanna prevent this for reinforced windows or not worry about it?

Reply:
I tried to avoid more changes then I needed to. If there wasn't a rwindow proc, I didn't add one